addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -391,11 +392,9 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
         //display rainbow at the end of any tour
         if (this.tours[tour_name].rainbowMan && this.running_tour !== tour_name &&
             this.tours[tour_name].current_step === this.tours[tour_name].steps.length) {
-            var $rainbow_message = $('<strong>' +
-                                '<b>Good job!</b>' +
-                                ' You went through all steps of this tour.' +
-                                '</strong>');
-            new RainbowMan({message: $rainbow_message}).appendTo(this.$body);
+            var rainbow_message = this.tours[tour_name].rainbowManMessage ||
+                _t('<strong><b>Good job!</b> You went through all steps of this tour.</strong>');
+            new RainbowMan({message: rainbow_message}).appendTo(this.$body);
```

Comment:
`const rainbowMessage` to follow our JS [guidelines](https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines) ;)
Or shorter:
```js
const message = ...
new RainbowMan({message}).appendTo(this.$body);
```

Reply:
done